pages/about-the-data.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import Head from 'next/head';
 import Primary from '../components/Primary';
-import Sidebar from '../components/Sidebar';
```

Comment:
We'd like to keep the sidebar on both the "About the Data" page and the "Related Organizations" page, and have it display the same information currently on the [About TJI](https://github.com/texas-justice-initiative/website-nextjs/blob/master/pages/about.js) page.